### DISCUSSION TITLE
Code  autocompetion for python is not working properly. please help.

### DISCUSSION THREAD

**Discussion by justinhskang** [LunarVim/LunarVim#4336] [Q&A]
Hi, 
I am a newbie to use lunarvim. I am trying to make the programming environment for python, and now I am troubling the lvim doesn't show the candidates for the code completion. It only shows buffer and snippets, and there is no LSP.

I checked pyright is installed from Mason. What is the problem?

BTY, I checked my code can attached to LSP as a client, but there is no listed configured servers from "LspInfo". Is this a problem?

thanks in advance.

**Comment by BtAbdul** [comment]
Here video tutorial by Chris@machine,
Video: https://youtu.be/lUssfjNzGNg?si=_yK2yIcED4UX9cAN
Blog post: https://betterprogramming.pub/lunarvim-as-a-python-ide-ca8f57ca9f3a
